prow/cmd/hook/main.go

```diff
@@ -132,6 +137,16 @@ func main() {
 		tokens = append(tokens, o.slackTokenFile)
 	}
 
+	// This is necessary since RocketChat is optional.
+	if o.rocketChatWebhookFile != "" {
+		tokens = append(tokens, o.rocketChatWebhookFile)
+	}
+
+	// This is necessary since RocketChat is optional.
+	if o.rocketChatChannel != "" {
+		tokens = append(tokens, o.rocketChatChannel)
```

Comment:
I'm not super familiar with Crier, but what stops us from adopting `o.rocketChat.Channel`, similar to `o.bugzilla.ApiKeyPath`? I.e., we should probably group the rocketChat-related things into `o.rocketChat`, no?

Reply:
I am not familiar with the `bugzilla` reporter, but my implementation is a copy of the `slack` reporter that does it the same way. Hope this is fine.